app/components/navigation/TopBar.tsx

```diff
@@ -83,13 +83,16 @@ export function TopBar({
         />
       </TouchableOpacity>
     ) : (
-      <ThemedIconButton
-        name={rightIcon}
-        onPress={rightAction}
-        size={30}
-        color={color}
-        testID={`topRightIcon-${rightIcon}`}
-      />
+      <TouchableOpacity onPress={rightAction}>
+        <View style={styles.defaultAvatar}>
+          <ThemedText
+            style={styles.avatarText}
+            testID={`topRightIcon-${rightIcon}`}
+          >
+            {title?.charAt(0).toUpperCase() || "A"} {/* Default letter */}
```

Comment:
Maybe use a different character, such as `?` or another special character.

Reply:
It's because it is the letter A of Anonymous!

Reply:
Sounds good to me, I'll approve the PR